### PR TITLE
feat(api): add workflow default tags after add repository webhook

### LINKED ISSUE #2448: Automatic workflow tag
Add git.branch; git.author as workflow tag when adding a repository webhook

### PR DESCRIPTION
**Pull request by bnjjj**
Signed-off-by: Benjamin Coenen <benjamin.coenen@hotmail.com>

1. Description
add workflow default tags after add repository webhook
1. Related issues
close #2448
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/api/migrate/default_tags.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	for _, proj := range projs {
+		workflows, err := workflow.LoadAll(db, proj.Key)
+		if err != nil {
+			log.Warning("DefaultTagsMigration> Cannot load all workflows for project %s : %s", proj.Key, err)
+			continue
+		}
+
+		for _, wf := range workflows {
+			tx, errTx := db.Begin()
+			if errTx != nil {
```
Comment (fsamin) [inline engine/api/migrate/default_tags.go]: don't you have to lock the workflow ?

Comment (bnjjj) [reply engine/api/migrate/default_tags.go]: I only lock the metadata of the workflow

Comment (fsamin) [reply engine/api/migrate/default_tags.go]: fine

**Comment by bnjjj** [comment]
tested on preprod with success

**Pull request merged**